apps/vsce/src/selectors/selectCodemodRunsTree.ts

```diff
@@ -10,16 +10,10 @@ export const selectCodemodRunsTree = (state: RootState, rootPath: string) => {
 		.filter(isNeitherNullNorUndefined)
 		.sort((a, b) => a.createdAt - b.createdAt)
 		.map((kase) => {
-			const label =
-				kase.codemodHashDigest !== undefined
-					? state.codemod.entities[kase.codemodHashDigest]?.name ??
-					  kase.codemodName
-					: kase.codemodName;
-
 			return {
 				node: {
 					hashDigest: kase.hash,
-					label,
+					label: kase.codemodName,
```

Comment:
What will happen when we run same codemod multiple times? Will it display all runs, or only 1?

Reply:
all runs

Reply:
<img width="500" alt="Screenshot 2024-04-04 at 16 59 35" src="https://github.com/codemod-com/codemod/assets/32841130/e163a979-7e47-4f26-b250-5c2758211b2f">
atm, we can distinguish via this